test/unit/export/service/ExportPackageTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoSyncServer\test\unit\export\service;
+
+use oat\generis\test\TestCase;
+use oat\taoSync\model\dataProvider\SyncDataProviderCollection;
+use oat\taoSync\package\SyncPackageService;
+use oat\taoSyncServer\export\service\ExportPackage;
+
+class ExportPackageTest extends TestCase
+{
+    public function testGetInvalidProvider()
```

Comment:
is it possible to write a real integration test?
something like https://github.com/oat-sa/extension-tao-delivery-rdf/pull/288/files#diff-8db4af812e97c0c9f760c40d3d46df91R61

Reply:
for this service, it is not needed

Reply:
it current stage no